misc/scripts/deploy-tools.sh

```diff
@@ -89,6 +94,49 @@ function deployplugin {
     done
 }
 
+function deploypluginwindows {
+    log "Compressing source into $PLUGIN_SRC_ZIP..."
+    cd $PLUGIN_SRC_DIR
+    rm -f $PLUGIN_SRC_ZIP
+    stashName=`git stash create`;
+    git archive --format zip --output $PLUGIN_SRC_ZIP $stashName
+
+    for ip in $IP_LIST
+    do
+        TARGET=root@$ip
+
+        log "Cleaning up older files from $TARGET..."
+        $SSH $TARGET <<-EOF
+            powershell Remove-Item -Recurse -Force $WIN_PLUGIN_SRC_DIR
```

Comment:
@shaominchen I'd probably leave those statements as is, as there's no other logging apart from that.

@msterin those warnings are because of the common `C` code being compiled by both `gcc` on Linux and `cl` on Windows. Let's discuss offline on how to go about fixing that.